pyomo/contrib/incidence_analysis/interface.py

```diff
@@ -208,56 +288,59 @@ def __init__(
                 )
             )
             self.var_index_map = ComponentMap(
-                    (var, i) for i, var in enumerate(self.variables))
+                (var, i) for i, var in enumerate(self.variables)
+            )
             self.con_index_map = ComponentMap(
-                    (con, i) for i, con in enumerate(self.constraints))
-            self.incidence_matrix = get_structural_incidence_matrix(
-                    self.variables,
-                    self.constraints,
-                    )
+                (con, i) for i, con in enumerate(self.constraints)
+            )
+            self.incidence_graph = get_bipartite_incidence_graph(
+                self.variables,
+                self.constraints,
+                # TODO: include_fixed=include_fixed?
```

Comment:
Just checking: a real TODO or residual note to yourself?

Reply:
Residual note. However, `include_fixed` is not necessary here, as we have already filtered above in the `_generate_variables_in_constraints` call.